src/api/public/apidocs-new/paths/build_project_repository_architecture_package_file_view_fileinfo.yaml

```diff
@@ -0,0 +1,50 @@
+get:
+  summary: This endpoint returns a specific artifact file contents
+  description: |
+    Given a project, repository, architecture and package, retrieve the given file's content.
+  operationId: getBuildProjectRepositoryArchitecturePackageFileViewFileinfo
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/repository_name.yaml'
+    - $ref: '../components/parameters/architecture_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - $ref: '../components/parameters/file_name.yaml'
+    - name: view
+      in: query
+      required: false
+      schema:
+        type: array
+        items:
+          type: string
+          enum:
+            - "fileinfo"
+            - "fileinfo_ext"
+  responses:
+    '200':
+      description:
+        Returns the artifact's actual file contents
```

Comment:
It doesn't return the content, its returning infos about the file, something like that: 
```
<fileinfo filename="hello-2.10-3.1.src.rpm">
  <name>hello</name>
  <version>2.10</version>
  <release>3.1</release>
  <arch>src</arch>
  <summary>A Friendly Greeting Program</summary>
  <description>The GNU hello program produces a familiar, friendly greeting.  It
allows nonprogrammers to use a classic computer science tool that would
otherwise be unavailable to them.  Because it is protected by the GNU
General Public License, users are free to share and change it.

GNU hello supports many native languages.</description>
  <size>765725</size>
  <mtime>1617270174</mtime>
  <requires>makeinfo</requires>
</fileinfo>
```

Reply:
Ouch, it's true. And it makes sense. Change